pixi/backend/api.js

```diff
@@ -38,6 +38,11 @@ const findAmpComponents = ($) => {
   return versionMap;
 };
 
+const isAmp = ($) => {
```

Comment:
We also want to check if it imports `.*\/v0.js`

Reply:
In the amp-toolbox the "isAmp" implementation just checks for the amp attribute.
What is the purpose of this additional check?
Should it be added with "and" or with "or"?

If it is "or" I think it is a very weak check to see if a page looks like it should be AMP since the pattern could match scripts that are not AMP easily. I think it would be better to check for `<style amp-` and `<script custom-element="amp-`

If it should be "and" I don't really see the benefit of adding it. When a user hosts the amp runtime on his server and the name no longer matches he should get an error that the page is not considered valid AMP and not that the page is no AMP at all (while the rest of the page might literally scream "I am an AMP page" ;-)

Reply:
Works for me.